src/Modules/Guilds/Hooks/useTimeDetail.ts

```diff
@@ -0,0 +1,51 @@
+import moment from 'moment';
+import { useTranslation } from 'react-i18next';
+import { ProposalState } from 'types/types.guilds.d';
+import {
+  getTimeDifferenceFromCurrentTimeHumanized,
+  isBeforeCurrentTime,
+} from 'utils/time/time';

```

Comment:
If you like you could add another `index.ts`file at the root of the `utils/time/index.ts` so you don't get this awkward pathway. We have done it for some `ActionBuilder` components.